tests/support/WebHelper.php

```diff
@@ -5,4 +5,8 @@
 
 class WebHelper extends \Codeception\Module
 {
+    public function changeBrowser($browser)
+    {
+        $this->getModule('WebDriver')->_reconfigure(['browser' => $browser]);
```

Comment:
It's ok with me, but I would like to pass it to you as you have a better idea about implementation.